multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -400,18 +421,22 @@ func (r *ServiceExportReconciler) updateSvcExportStatus(ctx context.Context, req
 
 	switch cause {
 	case serviceNotFound:
-		newCondition.Reason = getStringPointer("service_not_found")
-		newCondition.Message = getStringPointer("the Service does not exist")
+		newCondition.Reason = getStringPointer("ServiceNotFound")
```

Comment:
Could you also check if we add conditions for other resources, and use consistent Reason and Message format for all?

Reply:
Updated one in ResourceExport, others are expected format.